src/main/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClient.java

```diff
@@ -37,37 +35,34 @@
 
     public Long getCompanyTradedStatus(String companyNumber) {
         var getCompanyTradedStatusUrl = String.format("%s/company/%s/traded-status", oracleQueryApiUrl, companyNumber);
-        LOGGER.info(CALLING_ORACLE_QUERY_API_URL_GET, getCompanyTradedStatusUrl);
+        LOGGER.info(String.format(CALLING_ORACLE_QUERY_API_URL_GET, getCompanyTradedStatusUrl));
```

Comment:
If the URLs get changed in the future, these log lines could be missed when updating causing misleading log data.

Reply:
I think we'll be ok in this instance as that url string is used to make the real call, so if we do change it in future the log line should automatically pick it up. Perhaps remove the (get) from the CALLING_ORACLE_QUERY_API_URL_GET string.